### PR TITLE
Gulp task

### PR DESCRIPTION
**Pull request by scottnath**
Added a gulp task for importing patterns

@sergesemashko @beynya

### CODE REVIEW

**Comment by scottnath** [comment]
@beynya still free? can you look at this please?

**Comment by beynya** [comment]
@scottnath i'll take a look

File: gulp/patterns-import.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * @requires NPM:Gulp
+ * @requires NPM:lodash.merge
+ */
+module.exports = function (gulp, projectOptions) {
+
+  /* default options for pattern-importer */
+  var options = {
+    patternFiles: ['./app/bower_components/pattern-library/patterns/**/pattern.yml','./app/patterns-local/**/pattern.yml'],
+    patternImporterOptions: patternImporterUtils.getOptions()
+  }
```
Comment (beynya) [inline gulp/patterns-import.js]: add ";" please

**Comment by scottnath** [comment]
@beynya change complete. Any other issues you see?

**Comment by beynya** [comment]
looks good for me
@sergesemashko ?

**Comment by sergesemashko** [comment]
:+1:

**Pull request merged**